### PR TITLE
ARCSPC-942: Bug fix for focus indicators not functioning in Firefox

### PR DESCRIPTION
**Pull request by Hurlyburly2**
This bug stemmed from two issues, only one of which was Firefox dependent, and that was resolved by changing a change in the css. More pernicious was the expand buttons sometimes not receiving the actions that were being assigned with Javascript. This turned out to be because of the page loading order, and was resolved by creating a separate javascript file to embed on pages where the children_tree sidebar is used.

### CODE REVIEW

**Comment by ktamaral** [comment]
Looks good, but this branch may need to be re-based against development since there are some conflicts in public/assets/harvard.js

**Comment by pobocks** [comment]
I think I'll need to cut a new development and update master soon, I was
hoping to get the generalized table stuff merged to core first though.

File: public/assets/harvard-sidebar-expandbuttons.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        var $eadBtn = $("button.expandme");
+        if ($eadBtn.length > 0) {
+        $("button.expandme").focusin(function() {
+            $(this).find("i").addClass("focusIcon")
+        })
+        
+        $("button.expandme").focusout(function() {
+            $(this).find("i").removeClass("focusIcon")
+        })
+        // expandActionsAdded = true;
```
Comment (ktamaral) [inline public/assets/harvard-sidebar-expandbuttons.js:30]: Maybe remove this comment if it isn't needed?

**Pull request merged**